viewer/static/css/experiment-metadata.styl

```diff
@@ -0,0 +1,33 @@
+@import 'lib';
+
+$grid-padding = $min-padding;
+
+#experiment-metadata {
+  display: grid;
```

Comment:
Grid changes everything. It's not perfect (there's some really weird hoops you gotta jump through to make it wrap like Flexbox, for example) but it makes so many things so much easier. 😍